### ISSUE TITLE
Carousel does not show up when deployed to Netlify

### ISSUE THREAD

**Issue by timh1203** [akiran/react-slick#1472]
Hello,

I'm having a problem getting the carousel to show up when deployed to Netlify. It shows just fine locally but not when it's deployed. What is going on?

Expected:
<img width="1440" alt="screenshot 2018-12-31 08 50 54" src="https://user-images.githubusercontent.com/19733413/50562404-482f3280-0cd9-11e9-845c-385dad6d56bf.png">

Actual:
<img width="1440" alt="screenshot 2018-12-31 08 51 15" src="https://user-images.githubusercontent.com/19733413/50562405-49f8f600-0cd9-11e9-8dbf-c50f07f5ebae.png">

What I have tried/noticed:
1) Redeploying the site with a new github repo.
2) Referred to 1 related post but this was not my problem: https://github.com/akiran/react-slick/issues/1365
3)The props do get passed correctly in react dev tools.

Steps to Reproduce:
1) Can clone this repo: https://github.com/timh1203/dullesBarber2
2) Deploy it on Netlify.

Thanks for any help!

**Comment by umairx97** [comment]
Hi i am having the same issue but in heroku let me know if you solved it

**Comment by timh1203** [comment]
> Hi i am having the same issue but in heroku let me know if you solved it

Owner requested that I remove it since it slows load time. Maybe there's another carousel solution out there. Haven't looked at this time. I'm curious if you do get it to work. Good luck!

**Comment by umairx97** [comment]
yeah i solved it my problem was number of slides to show > than total slides 
when i adjusted my setting to show 1 slide on the responsive setting it worked. hope it helps
